lib/stat_tracker.rb

```diff
@@ -46,7 +41,24 @@ def create_game_teams
     @game_teams = GameTeam.from_csv(@game_teams_path)
   end
 
-  def all_teams
-    Team.all
+  def home_games
```

Comment:
as discussed, added methods to stat_tracker class that call methods housed in other classes